lib/diagnostic.js

```diff
@@ -46,10 +47,19 @@ let scopes = [/* your response here */];
 // method and temporarily bind it (or, change `this`) to `alderaan` before
 // executing it.
 //
+
+let result = deathStar.fire.call(alderaan);
 let fireOnAlderaan = function () {
-  // your code here
+  return result; // couldn't get it to run corectly inside of the function:(
 };
 
+//  Tried this but it didn't work :(
+//
+// let fireOnAlderaan = function () {
+//   deathStar.fire.call(alderaan);
```

Comment:
This _does_ work! You probably aren't passing the tests because you didn't return a value from `fireOnAlderaan`.